server/src/entities/asset.entity.ts

```diff
@@ -106,9 +106,6 @@ export class AssetEntity {
   @Column({ type: 'boolean', default: false })
   isExternal!: boolean;
```

Comment:
I noticed you removed this from the open API definition. Do we still need it here? If you end up making your change to have null libraryId indicate an upload library then this becomes redundant because you can just check if there's a library id present or not